examples/one-page.html

```diff
@@ -115,27 +114,43 @@
       }`;
 
       const Presentation = () => html`
-        <${Deck} theme=${theme} template=${template}>
+        <${Deck} theme=${theme} template=${template} transitionEffect="fade">
           <${Slide}>
             <${FlexBox} height="100%">
               <${SpectacleLogo} size=${500} />
             </${FlexBox}>
           </${Slide}>
           <${Slide}>
             <${FlexBox} height="100%" flexDirection="column">
-              <${Heading} fontSize="150px">SPECTACLE</${Heading}>
-              <${Heading} fontSize="h2">A ReactJS Presentation Library</${Heading}>
-              <${Heading} color="primary" fontSize="h3">Where you can write your decks in JSX, Markdown, or MDX!</${Heading}>
+              <${Heading} margin="0px" fontSize="150px">SPECTACLE</${Heading}>
+              <${Heading} margin="0px" fontSize="h2">A ReactJS Presentation Library</${Heading}>
+              <${Heading} margin="0px 32px" color="primary" fontSize="h3">Where you can write your decks in JSX, Markdown, or MDX!</${Heading}>
             </${FlexBox}>
             <${Notes}>
               <p>Notes are shown in presenter mode. Open up localhost:3000/?presenterMode=true to see them.</p>
             </${Notes}>
           </${Slide}>
-          <${Slide}>
+          <${Slide} transitionEffect="slide">
             <${Heading}>Code Blocks</${Heading}>
-            <${CodePane} fontSize=${18} language="cpp" autoFillHeight>
-              ${cppCodeBlock}
-            </${CodePane}>
+            <${Stepper} defaultValue=${[]} values=${[
```

Comment:
The one-page build script is doing some odd stuff here indentation-wise. I'll take a look.